terra_layer/permissions.py

```diff
@@ -4,3 +4,8 @@
 class LayerPermission(permissions.BasePermission):
     def has_permission(self, request, view):
         return request.user.has_perm('django_geosource.can_manage_sources')
+
+
+class ScenePermission(permissions.BasePermission):
+    def has_permission(self, request, view):
+        return request.user.has_perm('can_manage_layers')
```

Comment:
Create another permission is not needed, you can use the same as previously created (LayerPermission), juste change the permission checked that was a mistake.

Reply:
Done

Reply:
I do not understand your modifications, why do you still use Scene permission ?
That seems to be used only for safe_methods, but doesn't provide anything in other cases, so edition is not possible ?

Reply:
My bad, I fix this